src/main/groovy/com/stanfy/spoon/gradle/SpoonRunTask.groovy

```diff
@@ -146,11 +146,26 @@ class SpoonRunTask extends DefaultTask implements VerificationTask {
   }
 
   private static def findCpDependency(final Project project, final String prefix) {
-    def configuration = project.buildscript.configurations.classpath
-    return configuration.resolvedConfiguration.firstLevelModuleDependencies.find {
-      it.name.startsWith prefix
+        def configuration = project.buildscript.configurations.classpath
+        return configuration.resolvedConfiguration.firstLevelModuleDependencies.findResult {
```

Comment:
Also looks like if you changed `findChildDependency` a bit, it could be simplified:
```groovy
private static def findChildDependency(def dependencyList, final String prefix) {
  return dependencyList.findResult {
    if (it.name.startsWith(prefix)) {
      return it
    } else {
      return findChildDependency(it.children, prefix)
    }
  }
}
```
And no here you can simply do
```groovy
return findChildDependency(configuration.resolvedConfiguration.firstLevelModuleDependencies)
```

Haven't tested though :)